CogitoNTNU/web-backend
# feat: add endpoint to update image of members by sending a list of Images with correct names

### How to test:
Use this command in the terminal:

```bash
curl -X POST -F "images=@path_to_image.jpg" http://localhost:8000/api/member/image
```

Go to the swagger upload some images with names of Members and non members and check the Members afterwards
